apps/agent/buidler.config.js

```diff
@@ -0,0 +1,42 @@
+const { usePlugin } = require('@nomiclabs/buidler/config')
+const hooks = require('./scripts/buidler-hooks')
+
+usePlugin('@aragon/buidler-aragon')
```

Comment:
Rather than use the buidler-aragon plugin, I was wondering if we shouldn't move to a vanilla buidler set up for now? That way we might be able to use buidlerevm or other defaults rather than configure our own ganache chain / etc.

I'm not particularly concerned at the moment about providing a good frontend developer experience for others in these apps.

Reply:
As discussed offline, this is a good step forward.